projects/1.20/assets/my-nethers-delight/mynethersdelight/lang/zh_cn.json

```
@@ -0,0 +1,87 @@
+  "effect.mynethersdelight.b_pungent": "呛鼻",
+  "effect.mynethersdelight.g_pungent": "呛鼻",
+  "farmersdelight.tooltip.hot_cream": "烧尽状态效果",
+  "farmersdelight.tooltip.strider_egg": "清除1个负面效果",
+  "farmersdelight.tooltip.strider_feed.when_feeding": "喂给炽足兽时：",
+  "item.mynethersdelight.bleeding_tartar": "血淋塔塔",
+  "item.mynethersdelight.blue_tenderloin_steak": "诡蓝嫩烤腰肉排",
+  "item.mynethersdelight.breakfast_sampler": "早餐拼盘",
+  "item.mynethersdelight.chilidog": "火辣热狗",
+  "item.mynethersdelight.cooked_loin": "熟疣猪兽腰肉",
+  "item.mynethersdelight.crimson_stroganoff": "绯红菌奶油炖肉",
+  "item.mynethersdelight.fried_hoglin_chop": "炸疣猪兽排",
+  "item.mynethersdelight.hoglin_hide": "疣猪兽皮",
+  "item.mynethersdelight.hoglin_loin": "疣猪兽腰肉",
+  "item.mynethersdelight.hoglin_sausage": "疣猪兽香肠",
+  "item.mynethersdelight.hot_cream": "辛淇淋",
+  "item.mynethersdelight.hot_cream_cone": "辛淇淋填筒",
+  "item.mynethersdelight.hot_wings": "火辣鸡翅",
+  "item.mynethersdelight.hotdog": "热狗",
+  "item.mynethersdelight.minced_strider": "炽足兽肉末",
+  "item.mynethersdelight.plate_of_stuffed_hoglin": "盘装填馅疣猪兽",
+  "item.mynethersdelight.plate_of_stuffed_hoglin_ham": "盘装填馅疣猪兽火腿",
+  "item.mynethersdelight.plate_of_stuffed_hoglin_snout": "盘装填馅疣猪兽鼻",
+  "item.mynethersdelight.raw_stuffed_hoglin": "生填馅疣猪兽",
+  "item.mynethersdelight.roast_ear": "烤疣猪兽耳",
+  "item.mynethersdelight.roasted_sausage": "烤疣猪兽香肠",
+  "item.mynethersdelight.sausage_and_potatoes": "香肠配土豆",
+  "item.mynethersdelight.spicy_curry": "辛辣咖喱",
+  "item.mynethersdelight.spicy_hoglin_stew": "辛辣疣猪兽肉炖",
+  "item.mynethersdelight.spicy_skewer": "辛辣烤串",
+  "item.mynethersdelight.strider_egg": "炽足兽蛋",
+  "item.mynethersdelight.strider_rock": "炽足兽岩壳蛋",
+  "item.mynethersdelight.strider_slice": "炽足兽肉片",
+  "item.mynethersdelight.strider_stew": "炽足兽肉炖",
+  "item.mynethersdelight.strider_with_grilled_fungus": "炽足兽肉烤真菌",
+  "mynethersdelight.block.feast.use_knife": "你需要一把刀来切开它。",
+  "mynethersdelight.itemGroup.main": "My Nether's Delight",
+  "mynethersdelight.jei.forgoting": "忘却",
+  "mynethersdelight.jei.forgoting.accelerators": "周围有以下活化物时加快",
+  "mynethersdelight.jei.forgoting.fluid": "周围有熔岩时加快",
+  "mynethersdelight.jei.forgoting.light": "周围有以下火焰时加快",
+  "mynethersdelight.jei.forgoting.nether": "只能在下界忘却",
+  "mynethersdelight.jei.info.fungus_colony": "生长在复生沃土上的某些下界真菌会长成菌落。",
+  "mynethersdelight.jei.info.hoglin_hide": "优良的皮革来源，也可制作成精美的狩猎头颅挂饰。\n\n要是不想割坏皮革，可以试试看比剑更轻巧的武器……",
+  "mynethersdelight.jei.info.hot_cream": "它能烧尽当前的所有状态效果，并将它们转化成抗火和呛鼻。\n\n如果把这种膏状物放到脆皮筒里，那就只会烧掉一个效果了！炽足兽们似乎也更喜欢这种上菜形式……",
+  "mynethersdelight.jei.info.mushroom_colony": "生长在沃土或复生沃土上的某些蘑菇会长成菌落。",
+  "mynethersdelight.jei.info.plate_of_stuffed_hoglin": "如此原始的肉宴居然能分成这么多盘，真是奇妙……",
+  "mynethersdelight.jei.info.r_soil": "这种土壤能栽种和培育几乎所有的植物跟作物，润湿它需要的是火而不是水。",
+  "mynethersdelight.jei.info.strider_egg": "等下……炽足兽原来是卵胎生生物？不可思议……\n\n不管怎么说，只有小心谨慎地捕猎，才可能获得这种蛋。\n外壳很硬，也许只有破开才能吃了……",
+  "mynethersdelight.jei.info.wild_powdery": "烟药竹秆是绯红森林中极具侵略性的作物。\n\n它结出的浆果一触即爆，还是割断红色的茎好些……"

```

```suggestion
  "mynethersdelight.jei.info.wild_powdery": "烟药秆是绯红森林中极具侵略性的作物。\n\n它结出的浆果一触即爆，还是割断红色的茎好些……"
```
需要和修改后的 Powdery Cane 译名统一。

---

悲，这个地方提交出错了